app/logic/users.py

```diff
@@ -47,6 +48,19 @@ def removeUserInterest(program_id, username):
         interestToDelete.delete_instance()
     return True
 
+def getBannedUsers(program):
+    """
+    This function returns users banned from a program.
+    """
+    return ProgramBan.select().where(ProgramBan.program == program, ProgramBan.unbanNote == None)
```

Comment:
We are already getting what users are banned from a program above this in isElegibleForProgram. Why are you getting the users banned from a program again?

Reply:
isElegibleForProgram is for a single user whereas getBannedUsers is for all users. That way we can filter through a list in the table in jinja. It's querying just once instead of separate queries every time there's a user in the table.